flask_praetorian/utilities.py

```diff
@@ -105,3 +112,76 @@ def current_custom_claims():
     """
     jwt_data = get_jwt_data_from_app_context()
     return {k: v for (k, v) in jwt_data.items() if k not in RESERVED_CLAIMS}
+
+
+def deprecated(reason):
+    """
+    This is a decorator which can be used to mark functions
+    as deprecated. It will result in a warning being emitted
+    when the function is used.
+    """
+
+    if isinstance(reason, string_types):
+
+        # The @deprecated is used with a 'reason'.
```

Comment:
This would be a great bit of documentation to add to the docstring for the decorator

Reply:
submitted